xmpp/xmpp.php

```diff
@@ -94,7 +94,7 @@ function xmpp_addon_settings(App $a, &$s)
 
 function xmpp_login()
 {
-	if (!$_SESSION["allow_api"]) {
+	if (isset($_SESSION["allow_api"]) && !$_SESSION["allow_api"]) {
```

Comment:
This can be simplified to `!empty($_SESSION["allow_api"])`.

Reply:
Indeed this ```empty``` would be the better way - because this value is only set to "true" but never to false in the code. So my code won't work at all.